### ISSUE TITLE
HealthRecords - Medications 

### ISSUE THREAD

**Issue by abuckley3385** [department-of-veterans-affairs/va.gov-team#19588]
## Product Documentation
- [Questionnaires Product Outline ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/product-outline.md)
- [HealthRecord Initiative](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/initiatives/health-record.md)

## User Story/ies

**(5F) As a Veteran, I want to update my current medications so that my care team is aware of what Im currently taking and need refills so that Im able to fill RX when needed.**

Review and Edit
(2A) As a Veteran, I want to review and update my health information to ensure its accuracy, so my care team can focus on my treatment plan and whats important to me vs. data entry during my visit.

Since Last Time
(2B) As a Veteran, I want to only provide updates to my health information that is affecting me now, changed since last time, or due for screening, so that I save time documenting my information and my care team can focus on the changes to my treatment plan and whats important to me.

Patient-centric plain language
(3A) As a Veteran, I want to have complicated medical terms explained to me in a tone and language that I understand, so that I am able to provide accurate updates to my health information and be prepared to ask questions to my care team for further education.
(5A) As a Veteran, I want to provide my health status and updates to my medical history so that I do not have to feel I have to repeat the information to my nurse and provider in the same and subsequent visits.

---
## Tasks
- [ ] [Medications Feature Outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/questionnaire/product/features/health-record/medications.md)
- [ ] User Research 
- [ ] Mockups link 

## Acceptance Criteria
- [ ] UI review with entire team for feature capabilities

**Comment by julieadair1** [comment]
@jparker0425 this looks mostly complete.  Can we close the epic?
cc: @batemapf @kristenmcconnell

**Comment by kristenmcconnell** [comment]
On all three of these Health Record epics, we did the Veteran research and had a pretty good understanding of what Veterans wanted. We also looked into what was available on MyHealtheVet, Lighthouse, and something else that I can't remember right now. We had some pretty detailed wireframes that we usability tested. We just needed to talk to clinicians to find out what information was required when a Veteran entered in a new medication, allergy, or immunization. We probably could get this from Vista engineers too. We wanted to look into Cerner as well. We also want to check with clinicians on how they wanted Veterans to confirm that each of the lists were accurate. That is the high-level summary of what we did and how far we got.
